attribute_set_completeness/models/attribute_set_owner_mixin.py

```diff
@@ -29,24 +27,33 @@ class AttributeSetOwnerMixin(models.AbstractModel):
 
     @api.multi
     def _compute_completion_rate(self):
-        for record in self:
-            attribute_set_id = record.attribute_set_id
-            completion_config = attribute_set_id.attribute_set_completeness_ids
-            if completion_config:
-                completion_rate = 0.0
-                not_complete_crit = self.env["attribute.set.completeness"].browse()
-                for criteria in completion_config:
-                    field_name = criteria.field_id.name
-                    if record[field_name]:
-                        completion_rate += criteria.completion_rate
+        for rec in self:
```

Comment:
@clementmbr Thak you for theses changes. For better readability I would compute `is_completion_rate_visible` and `is_completion_state_visible` into 2 distinct methods `_compute_is_completion_rate_visible` and `_compute_is_completion_state_visible` With your current approach, it's not possible to easily understand the criteria that makes these fields visible or not.